.github/workflows/sync-mcp-registry.yml

```diff
@@ -0,0 +1,66 @@
+name: Sync MCP Registry
+
+on:
+  schedule:
+    # Run daily at 2 AM UTC
+    - cron: '0 2 * * *'
+  workflow_dispatch:  # Allow manual trigger
+
+permissions:
+  contents: write  # Required to commit changes
+  issues: write    # Required to create issues
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      
+      - name: Install dependencies
+        run: |
+          pip install requests pyyaml packaging openai
```

Comment:
can we get a requirements.txt instead? would make this a little easier to work with